flytekit/deck/deck.py

```diff
@@ -174,3 +174,39 @@ def get_deck_template() -> "Template":
         autoescape=select_autoescape(enabled_extensions=("html",)),
     )
     return env.get_template("template.html")
+
+
+class PythonDependencyDeck(Deck):
+    """
+    PythonDependencyDeck is a deck that contains information about the library name and version.
+    """
+
+    def __init__(self, name: Optional[str] = "Python Dependency", html: Optional[str] = ""):
+        super().__init__(name, html)
+
+    @property
+    def html(self) -> str:
+        try:
+            from flytekitplugins.deck.renderer import TableRenderer
+        except ImportError:
+            warning_info = "Plugin 'flytekit-deck-standard' is not installed. To display python dependency, install the plugin in the image."
+            logger.warning(warning_info)
+            return warning_info
+
+        import subprocess
+
+        import pandas as pd
+
+        try:
+            installed_packages = subprocess.check_output(["pip", "freeze"]).decode().split("\n")
+        except subprocess.CalledProcessError as e:
+            logger.error(f"Error occurred while fetching installed packages: {e}")
+            return ""
+
+        columns_name = ["Library", "Version"]
+        installed_packages = [package.split("==") for package in installed_packages if package]
+        df = pd.DataFrame(installed_packages, columns=columns_name)
+        html = TableRenderer().to_html(df, header_labels=columns_name)
```

Comment:
how about we simplify this to:
```suggestion
        try:
            installed_packages = json.loads(subprocess.check_output(["pip", "list", "--format", "json"]))
        except subprocess.CalledProcessError as e:
            logger.error(f"Error occurred while fetching installed packages: {e}")
            return ""

        df = pd.DataFrame(installed_packages)
        html = TableRenderer().to_html(df)
```

Reply:
updated

Reply:
I like this simple way